packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -383,6 +408,30 @@ function Menu(props: MenuProps): React.ReactElement {
 		setSelection({ type: "telemetryMenuSelection" });
 	}
 
+	function handleThemeChange(
+		event,
+		option: {
+			optionValue: string | undefined;
+			optionText: string | undefined;
+			selectedOptions: string[];
+		},
+	): void {
+		switch (option.optionText) {
```

Comment:
Is it possible to check for some `value` attribute of the options, like it's usually done with regular HTML dropdowns? Preferable to check against something like that which would probably be more stable than the text values in the dropdown. Also works around any future issues with localization of the texts :).